src/main/java/lotto/controller/JavaLotto.java

```diff
@@ -19,18 +20,18 @@ public class JavaLotto {
 
     public void playLotto() {
 
-        int amount = inputView.inputPurchaseAmountGuide();
+        PurchaseInfo purchaseInfo
+                = new PurchaseInfo(inputView.inputPurchaseAmountGuide(),inputView.inputManualLottoPurchaseAmount());
 
-        Lottos lottos = Lottos.createLottos(amount, LottoShuffleGenerator.getLottoShuffleGenerator());
+        Lottos lottos = inputView.inputManualLottoGuide(purchaseInfo);
+        lottos.additionalLottos(Lottos.createLottos(purchaseInfo, LottoShuffleGenerator.getLottoShuffleGenerator()));
 
-        resultView.NumberOfLotto(lottos.getNumberOfLotto());
-        resultView.printPurchasedLottos(lottos);
+        resultView.printPurchaseLottoResult(lottos);
 
         LottoResultStatistic lottoResultStatistic
```

Comment:
인자로 전달하는 대상의 경우 인라인으로 처리하기보다 변수선언을 하여 표현하는것이
가독성 측면에서 좋다고 생각해요 🙂

```suggestion
              WinningLotto winningLotto = new WinningLotto(inputView.inputWinnerNumber(), inputView.inputBonusNumber());
      LottoResultStatistic lottoResultStatistic = lottos.getResultStatistic(winningLotto);
```